lib/src/runner/browser/browser_manager.dart

```diff
@@ -225,7 +225,7 @@ class BrowserManager {
           sink.close();
         }));
 
-    return await _pool.withResource(() async {
+    return await _pool.withResource/*<Future<RunnerSuite>>*/(() async {
```

Comment:
Ouch! If you want to add a bug note, this one is covered by:
https://github.com/dart-lang/sdk/issues/27284